build.gradle

```diff
@@ -3,13 +3,31 @@ description = 'Opencensus Proto'
 apply plugin: 'idea'
 apply plugin: 'java'
 apply plugin: 'com.google.protobuf'
+apply plugin: 'maven'
+apply plugin: "signing"
+
+group = "io.opencensus"
+version = "0.0.1-SNAPSHOT" // CURRENT_OPENCENSUS_PROTO_VERSION
+
+sourceCompatibility = 1.6
```

Comment:
Just a minor nit here, I have just filed https://github.com/census-instrumentation/opencensus-java/issues/1428 while would require us to update the compatibilities to 1.7 since the Java OpenCensus code uses features present in only 1.7

Reply:
Nevermind this comment.